### ISSUE TITLE
Commit cherry pick

### ISSUE THREAD

**Issue by timsofteng** [extrawurst/gitui#493]
Hello. Could you explain please how can I cherry pick commit from one branch to another?

Thanks!

**Comment by extrawurst** [comment]
Hi @timsofteng thanks for your interest in gitui. *Cherry picking* is not yet implemented in gitui, so you have to fallback to the regular git cli: https://git-scm.com/docs/git-cherry-pick

**Comment by timsofteng** [comment]
@extrawurst Got it. Please consider to implement this feature. I've tried it in lazygit and it's very useful.
Thanks.

**Comment by extrawurst** [comment]
It is definitely a good feature, but not on my high prior's list (see [ROADMAP](https://github.com/extrawurst/gitui#roadmap-to-10))
I am open for Pull Requests :)

**Comment by timsofteng** [comment]
Hello. Any updates in this direction?

**Comment by extrawurst** [comment]
nope, this is not on my personal roadmap to 1.0 - still open to PRs

**Comment by Lyahi** [comment]
I miss this option in gitui. Dev, please consider adding this option. ;)

**Comment by searleser97** [comment]
+1 to this feature, please consider it for v1, it is quite useful feature

**Comment by xinglinsky** [comment]
need this feature +1

**Comment by kazuhideoki** [comment]
need this feature +1.
I think it would be quite useful.

**Comment by wit-l** [comment]
need this feature +1

**Comment by extrawurst** [comment]
ideally you thumbs up the original issue instead of comments that do not add discussion content 👍
